packages/react/src/compass/components/root-operation/root-operation.tsx

```diff
@@ -27,26 +65,69 @@ export const RootOperation = ({
     );
   }
 
+  const virtualItems = virtualizer.getVirtualItems();
+
   return (
-    <ul className={rootOperationClass}>
-      {Object.keys(fields)
-        .sort()
-        .map((field) => (
-          <Field
-            key={field}
-            ancestors={[
-              ...ancestors,
-              {
-                type: 'FIELD',
-                field: fields[field],
-                selection:
-                  operationDefinition?.selectionSet?.selections.find((s) =>
-                    'name' in s ? s.name.value === fields[field].name : false,
-                  ) || null,
-              },
-            ]}
+    <div className={rootOperationStyles.container}>
+      <div className={rootOperationStyles.searchContainer}>
```

Comment:
Let's remove this search and come back to it after we've had the time to think about design.